esm/package.json

```diff
@@ -0,0 +1,3 @@
+{
+  "module": "index.js"
```

Comment:
Same as before. We need to check .mjs files for default behavior. Can you please add a new folder to handle esm + js extension + module field?

Reply:
Lets skip that for now, feels like very much of an edge case